<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ToDo</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>40</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">Background:rgb(40, 149, 34);Color:black;border-radius: 5px; border-style: outset;border-width: 2px;
</string>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>120</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">Background:rgb(140, 19, 10);Color:black;border-radius: 5px; border-style: outset;border-width: 2px;</string>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>491</width>
      <height>341</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 2px; border-width: 1px;border-color:rgb(53, 83, 255); border-style: outset;
background-color:rgb(162, 162, 162)</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideNone</enum>
    </property>
    <widget class="QWidget" name="tab_4">
     <property name="styleSheet">
      <string notr="true">background-color:rgb(162, 162, 162)</string>
     </property>
     <attribute name="title">
      <string>All tasks</string>
     </attribute>
     <widget class="QTableView" name="tableAll">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>481</width>
        <height>311</height>
       </rect>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>77</number>
      </attribute>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <property name="styleSheet">
      <string notr="true">background-color:rgb(162, 162, 162)</string>
     </property>
     <attribute name="title">
      <string>Active tasks</string>
     </attribute>
     <widget class="QTableView" name="tableActive">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>481</width>
        <height>311</height>
       </rect>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>77</number>
      </attribute>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <property name="styleSheet">
      <string notr="true">background-color:rgb(162, 162, 162)</string>
     </property>
     <attribute name="title">
      <string>Failed tasks</string>
     </attribute>
     <widget class="QTableView" name="tableFailed">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>481</width>
        <height>311</height>
       </rect>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>77</number>
      </attribute>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <property name="styleSheet">
      <string notr="true">background-color:rgb(162, 162, 162)</string>
     </property>
     <attribute name="title">
      <string>Complited tasks</string>
     </attribute>
     <widget class="QTableView" name="tableComplited">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>481</width>
        <height>311</height>
       </rect>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>77</number>
      </attribute>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionAbout_add"/>
    <addaction name="separator"/>
    <addaction name="actionAbout_delete"/>
    <addaction name="separator"/>
    <addaction name="actionAbout_edit"/>
    <addaction name="separator"/>
    <addaction name="actionAbout_mark"/>
   </widget>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout_add">
   <property name="text">
    <string>About add</string>
   </property>
  </action>
  <action name="actionAbout_delete">
   <property name="text">
    <string>About delete</string>
   </property>
  </action>
  <action name="actionAbout_edit">
   <property name="text">
    <string>About edit</string>
   </property>
  </action>
  <action name="actionAbout_mark">
   <property name="text">
    <string>About mark</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
